src/abstract_test_client.rs

```diff
@@ -187,49 +187,20 @@ impl TestClient {
                     ),
                 },
             },
-        );
-
-        if result.is_ok() {
-            let provider = self.provider(Opcode::PsaGenerateKey);
-            let auth = self.auth.bytes().to_vec();
-
-            if let Some(ref mut created_keys) = self.created_keys {
-                let _ = created_keys.insert((key_name, auth, provider));
-            }
-        }
```

Comment:
I'm guessing `generate_key` already does this?

Reply:
Yes 😺